tests/kuttl/test-suites/default/deps/infra.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: core.openstack.org/v1beta1
+kind: OpenStackControlPlane
+metadata:
+  name: openstack
+spec:
+  storageClass: crc-csi-hostpath-provisioner
+  mariadb:
+    enabled: false
+  galera:
+    enabled: true
+    templates:
+      openstack:
+        storageRequest: 500M
+        storageClass: crc-csi-hostpath-provisioner
+      openstack-cell1:
+        storageRequest: 500M
+        storageClass: crc-csi-hostpath-provisioner
```

Comment:
we do not need a sereate cell galara cluster in this test env, partly because we wont be deploying nova.